.github/ISSUE_TEMPLATE/2_new-hub-collect-info.yml

```diff
@@ -112,53 +124,59 @@ body:
         - **Specific cloud provider or datacenter**: If you have restrictions about which cloud provider to use, and which data center the hub should live in (e.g. if it is next to a specific dataset).
         - **Dedicated Scalable Dask Cluster** with [Dask Gateway](https://gateway.dask.org/): if you have very large datasets or otherwise need high-performance parallelizable computing via the cloud.
         - **Dedicated Kubernetes cluster**: If you need to run the infrastructure on your own cloud account, or otherwise wish to have a dedicated cluster.
-      value: |
-        - [ ] Specific cloud provider or datacenter: <!-- Specify which if yes -->
-        - [ ] Dedicated Kubernetes cluster
-        - [ ] Scalable Dask Cluster
+      multiple: true
+      options:
+        - Specific cloud provider or datacenter (otherwise GCP)
+        - Dedicated Kubernetes cluster
+        - Scalable Dask Cluster
```

Comment:
I feel like this section would work better as checkboxes, rather than a dropdown

<img width="1072" alt="Screenshot 2022-10-07 at 13 55 46" src="https://user-images.githubusercontent.com/44771837/194559217-0d91d153-c2dc-4f65-bcec-d8cb0517b713.png">

Reply:
good idea, yes! I've just added a commit that turns this into checkboxes. TY!